.blazar.yaml

```diff
@@ -9,6 +9,12 @@ buildpack:
 # Make sure to also check hbase-assembly/.blazar.yaml to see if any changes should be added there as well.
 # Prefer making changes to prepare_environment.sh instead, if possible.
 
+env:
+  MAVEN_ARGS: ""
```

Comment:
Why do these need to be defined empty here?

Reply:
We're only letting `write-build-env-var` to modify env variables explicitly configured in the build environment. We might have to restrict `write-build-env-var` even further in the future to only modifying env variables defined in a buildpack instead of `.blazar.yaml` but for now this makes modifiable env variables as part of a build config explicit and we have a record of all env variables a build is modifying on our end

Reply:
Can you add a comment to that effect? Otherwise it's not inherently obvious. After that I can merge this. Same goes for the hadoop PR

Reply:
How does the comment read to you?

Reply:
Looks good, thanks!